Update manage_users.md (#716)
SEKOIA-IO/documentation: docs/getting_started/manage_users.md

# Manage users

## Overview

> Admins or user with the right permissions are able to add users with a role.

Reviewer: ```suggestion
Admins and users with the right permissions are able to add users with a role.
```

A role has attached permissions allowing a user to access, view pages and use its features.

> In the following sections, we describe how to set and use to manage your users.

Reviewer: ```suggestion
In the following sections, you will learn how to manage your users.
```
